src/components/package/edit-managed/managed-package-edit.js

```diff
@@ -83,9 +82,9 @@ class ManagedPackageEdit extends Component {
 
   handleSelectionToggle = (e) => {
     if (e.target.checked) {
+      this.props.change('allowKbToAddTitles', true);
```

Comment:
don't you also need to roll this back in `cancelSelectionToggle`?

Reply:
🤔 Hm, I don't think so. That is just to decide if you want to remove the package entirely. This modal: 
![image](https://user-images.githubusercontent.com/2072894/40744881-8817c770-641b-11e8-93c3-21f19f1cefb4.png)

If you click yes it'll remove it and `allowKbToAddTitles` is irrelevant. If you click no then nothing changed :)